reviews/config.py

```diff
@@ -7,6 +7,7 @@
 # Github Config
 GITHUB_TOKEN = config("GITHUB_TOKEN", cast=str, default="")
 GITHUB_USER = config("GITHUB_USER", cast=str, default="")
+GITHUB_URL = config("GITHUB_USER", cast=str, default="https://api.github.com")
```

Comment:
should this be `config("GITHUB_URL", ...)`?

Reply:
Good catch :+1: ill fix this